### PR TITLE
Zewaka's Readme command

### PR DESCRIPTION
**Pull request by TheTyrant25**
Using [readme should now yield a snappy message and image expressing disdain at people ignoring the code guides instead of having to say it over and over.

This is my first time pull requesting to gooncogs so i'd appreciate any double checking and reviewing.

### CODE REVIEW

**Comment by pali6** [comment]
the uploading code is wrong but either way I'd probably rather have it just include a link instead of uploading the picture every time
Just like upload the picture to discord and include its url

**Comment by TheTyrant25** [comment]
gotcha

File: goonmisc/goonmisc.py [hunk 1]
```diff
@@ -900,3 +900,9 @@ async def close_menu(
     ):
         with contextlib.suppress(discord.NotFound):
             await message.delete()
+
+    @commands.command(aliases=["readme"])
```
Comment (pali6) [inline goonmisc/goonmisc.py]: There's no point in adding the same alias as is the name of the command
```suggestion
    @commands.command()
```

File: goonmisc/goonmisc.py [hunk 2]
```diff
@@ -900,3 +900,9 @@ async def close_menu(
...
+    async def readme(self, ctx: commands.Context,):
```
Comment (pali6) [inline goonmisc/goonmisc.py]: invlaid syntax
```suggestion
    async def readme(self, ctx: commands.Context):
```

File: goonmisc/goonmisc.py [hunk 3]
```diff
@@ -900,3 +900,9 @@ async def close_menu(
...
+        ctx.send("Users are reminded that the official code guides and readmes exist for a *reason*, \
```
Comment (pali6) [inline goonmisc/goonmisc.py:907]:
```suggestion
        """Shows a passive aggressive message about how users should read the guides."""
        ctx.send("Users are reminded that the official code guides and readmes exist for a *reason*, \
```

**Comment by pali6** [comment]
I personally think this is too passive aggressive and also the name `readme` is too generic which will lead to people trying it just out of curiosity and be greeted with a hostile message. But if zewaka requested it then shrug

**Pull request merged**